src/api/store/message.py

```diff
@@ -38,16 +42,55 @@ def get_logo(id):
         if com:
             return com.logo.file.url if com.logo else None
         
-    
 
-
-def get_message_recipients(audience, audience_type):
-    if not is_null(audience):
+def get_message_recipients(audience, audience_type, community_ids,sub_audience_type):
+    
```

Comment:
can we add some description for what this function does?

Reply:
Added